api.planx.uk/modules/analytics/metabase/collection/types.ts

```diff
@@ -0,0 +1,54 @@
+import { z } from "zod";
+import type { ValidatedRequestHandler } from "../../../../shared/middleware/validate.js";
+
+type ApiResponse<T> = {
+  data?: T;
+  error?: string;
+};
+
+export const checkCollectionsSchema = z.object({
+  query: z.object({
+    name: z.string().min(1, "Name parameter is required"),
+  }),
+});
+
+export type CheckCollectionsRequest = ValidatedRequestHandler<
```

Comment:
```suggestion
export type CheckCollections = ValidatedRequestHandler<
```

nit: These aren't requests, but request handlers - `ValidatedRequestHandler<Req, Res>`